runelite-client/src/main/java/net/runelite/client/plugins/microbot/playerassist/combat/BuryScatterScript.java

```diff
@@ -0,0 +1,38 @@
+package net.runelite.client.plugins.microbot.playerassist.combat;
+
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.client.plugins.microbot.Microbot;
+import net.runelite.client.plugins.microbot.Script;
+import net.runelite.client.plugins.microbot.playerassist.PlayerAssistConfig;
+import net.runelite.client.plugins.microbot.util.inventory.Rs2Inventory;
+import net.runelite.client.plugins.microbot.util.inventory.Rs2Item;
+import net.runelite.client.plugins.microbot.util.player.Rs2Player;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+public class BuryScatterScript extends Script {
+public boolean run(PlayerAssistConfig config) {
+    mainScheduledFuture = scheduledExecutorService.scheduleWithFixedDelay(() -> {
+        try {
+            if (!Microbot.isLoggedIn() || !super.run() || (!config.toggleBuryBones() && !config.toggleScatter())) return;
+
+            processItems(config.toggleBuryBones(), Rs2Inventory.getBones(), "bury");
```

Comment:
with following config it doesn't pick up the bones

![image](https://github.com/chsami/microbot/assets/23292806/bb60641b-6bca-42c7-a377-346703427959)


![image](https://github.com/chsami/microbot/assets/23292806/7885b662-a770-4afd-9749-23de1519b5d9)

Reply:
Fixed, if bury is enabled and loot is enabled it will loot regardless of value.
Same with scatter.